lms/util/lti_launch.py

```diff
@@ -1,7 +1,11 @@
 """Decorator that add lti validation capabilities to a pyramid view."""
+from __future__ import unicode_literals
+
 import pylti.common
+from pyramid.i18n import TranslationString as _
```

Comment:
I realise it's not there on master either, but we should put a blank line after these third-party imports before our own imports. I'm not sure why `make lint` isn't catching this, actually.